astrosat_users/forms/forms_auth.py

```diff
@@ -67,6 +67,7 @@ class RegistrationForm(AllAuthRegistrationForm):
     field_order = ["email", "password1", "password2", "accepted_terms"]
 
     accepted_terms = forms.BooleanField(label="Accept Terms & Conditions")
+    pending_customer = forms.BooleanField(label="User is Registering as part of a Customer", required=False)
```

Comment:
Why does this this field not appear in the above `field_order`, like `accepted_terms` is?

Reply:
Because I forgot.  Good catch.

Reply:
Done in 12c396f3af144df369b9292f23b70024fc83e707